Spigot-Server-Patches/0705-OfflinePlayer-advancements.patch

```diff
@@ -0,0 +1,76 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Jake Potrebic <jake.m.potrebic@gmail.com>
+Date: Thu, 22 Apr 2021 01:41:56 -0700
+Subject: [PATCH] OfflinePlayer advancements
+
+
+diff --git a/src/main/java/net/minecraft/server/AdvancementDataPlayer.java b/src/main/java/net/minecraft/server/AdvancementDataPlayer.java
+index 7a8a1960882e291c46301d07da3e1c5415516893..de92b184ce2d74d9216b88f66207d2752620a61f 100644
+--- a/src/main/java/net/minecraft/server/AdvancementDataPlayer.java
++++ b/src/main/java/net/minecraft/server/AdvancementDataPlayer.java
+@@ -225,6 +225,7 @@ public class AdvancementDataPlayer {
+         this.b(advancementdataworld);
+     }
+ 
++    public void save() { b(); } // Paper - OBFHELPER
+     public void b() {
+         if (org.spigotmc.SpigotConfig.disableAdvancementSaving) return; // Spigot
+         Map<MinecraftKey, AdvancementProgress> map = Maps.newHashMap();
+diff --git a/src/main/java/net/minecraft/server/players/PlayerList.java b/src/main/java/net/minecraft/server/players/PlayerList.java
+index 2df8e914f66176e22aeddf8b94a83af5ea921d88..ebdddc5ed059e3e714677926986667701fcfbbea 100644
+--- a/src/main/java/net/minecraft/server/players/PlayerList.java
++++ b/src/main/java/net/minecraft/server/players/PlayerList.java
+@@ -1460,6 +1460,20 @@ public abstract class PlayerList {
+         advancementdataplayer.a(entityplayer);
+         return advancementdataplayer;
+     }
++    // Paper start
++    public AdvancementDataPlayer getAdvancementData(UUID uuid) {
```

Comment:
is this code copied form somewhere? if so, please note that. if not, I would like some nicer local var names for the files and obfhelpers for server.a and advancementDataPlayer.a